_data/projects/Maven-Using-CMD.yml

```diff
@@ -0,0 +1,12 @@
+name: Maven-Using-CMD
+desc: Creating a Hello World code project in Java using the Apache Maven build tool and automating it using Jenkins.
+site: https://github.com/NishkarshRaj/Maven-Using-CMD
+tags:
+- Maven
```

Comment:
@NishkarshRaj we're almost there. We made a recent change to validate all tags are lower-case, and these need to be changed to make CI pass:

```
Tag 'Maven' contains uppercase characters
Tag 'Java' contains uppercase characters
Tag 'XML' contains uppercase characters
Tag 'Jenkins' contains uppercase characters
Tag 'Automation' contains uppercase characters
```

Reply:
@shiftkey I have made the specified changes, please review. :smile: